_topic_map.yml

```diff
@@ -616,6 +616,9 @@ Topics:
   - Name: Sysctls
     File: sysctls
     Distros: openshift-origin,openshift-enterprise
+  - Name: Encrypting Data at Rest
```

Comment:
This is not about REST but about "at rest". I'll try to find a better wording to make it clear.